src/deployment-guides/tko-aws-airgap.md

```diff
@@ -0,0 +1,348 @@
+# Deploy Tanzu for Kubernetes Operations on AWS in an Airgapped Environment
+
+This document outlines the steps for deploying VMware Tanzu for Kubernetes Operations on AWS in an airgapped (internet-restricted) environment. The deployment is based on the reference design provided in [VMware Tanzu Kubernetes Grid on AWS Airgap Reference Design](../reference-designs/tko-on-aws-airgap.md).
+
+## Prerequisites
+Before deploying VMware Tanzu for Kubernetes Operations in an AWS airgapped environment, ensure that the following are set up.
+
+* **AWS Account**: An IAM user account with **administrative privileges**.
+
+* **AWS Resource Quotas**: Sufficient quotas to support both the management cluster and the workload clusters in your deployment. Otherwise, the cluster deployments will fail. Depending on the number of workload clusters you plan to deploy, you may need to increase the AWS services quotas from their default values. You will need to increase the quota in every region in which you deploy Tanzu Kubernetes Grid.
```

Comment:
```suggestion
* **AWS Resource Quotas:** Sufficient quotas to support both the management cluster and the workload clusters in your deployment. Otherwise, the cluster deployments will fail. Depending on the number of workload clusters you plan to deploy, you may need to increase the AWS services quotas from their default values. You will need to increase the quota in every region in which you deploy Tanzu Kubernetes Grid.
```